tests/specs/api.e2e.js

```diff
@@ -64,7 +67,7 @@ describe('newrelic api', () => {
     })
   })
 
-  describe('noticeError api', () => {
+  describe('noticeError()', () => {
```

Comment:
Keep it consistent ;)

Reply:
Do you mean you'd prefer to leave these as `... api` instead of `()`?

Reply:
No, you missed one.

Reply:
Maybe not. I might not be reading the diff correctly.